elf/elf.go

```diff
@@ -460,6 +460,7 @@ func (b *Module) Load(parameters map[string]SectionParams) error {
 	if err != nil {
 		return err
 	}
+	defer b.file.Close()
```

Comment:
Right, that's like a virtual in-memory File, so I don't think we need to close it.